.github/workflows/build.yml

```
@@ -344,7 +344,9 @@ jobs:
       - name: Run tests
         env:
           RELEASE_TAG: ${{ github.event.release.tag_name }}
-        run: pytest --runslow
+        run: |
+          export PYTHONPATH=$(pwd)/src
+          pytest --runslow
```

With my other review comments in mind:

```suggestion
        run: pytest
```

---

Addressed here https://github.com/cisagov/con-pca-api/pull/632/commits/c81fbad36e55f11e0499d23e61c014ef5bcfb877